specification/web/resource-manager/readme.md

```diff
@@ -97,59 +97,709 @@ directive:
   approved-by: "@ravbhatnagar"
 ```
 
+### Tag: package-2018-11-only
+
+These settings apply only when `--tag=package-2018-11-only` is specified on the command line.
+
+``` yaml $(tag) == 'package-2018-11-only'
+input-file:
+- Microsoft.Web/stable/2018-11-01/Certificates.json
+directive:
+  # suppress each RPC 3019 error
+- where: $.definitions.Identifier.properties
+  suppress: R3019
+  reason: It's an old API, will resolve in next API version
+  approved-by: "@ravbhatnagar"
+```
 
 ### Tag: package-2018-02
 
-These settings apply only when `--tag=package-2018-02` is specified on the command line.
+These settings apply only when `--tag=package-2018-02` is specified on the command line.
+
+``` yaml $(tag) == 'package-2018-02'
+input-file:
+- Microsoft.CertificateRegistration/stable/2018-02-01/AppServiceCertificateOrders.json
+- Microsoft.CertificateRegistration/stable/2018-02-01/CertificateRegistrationProvider.json
+- Microsoft.DomainRegistration/stable/2018-02-01/Domains.json
+- Microsoft.DomainRegistration/stable/2018-02-01/TopLevelDomains.json
+- Microsoft.DomainRegistration/stable/2018-02-01/DomainRegistrationProvider.json
+- Microsoft.Web/stable/2018-02-01/Certificates.json
+- Microsoft.Web/stable/2018-02-01/CommonDefinitions.json
+- Microsoft.Web/stable/2018-02-01/DeletedWebApps.json
+- Microsoft.Web/stable/2018-02-01/Diagnostics.json
+- Microsoft.Web/stable/2018-02-01/Provider.json
+- Microsoft.Web/stable/2018-02-01/Recommendations.json
+- Microsoft.Web/stable/2018-02-01/ResourceProvider.json
+- Microsoft.Web/stable/2018-02-01/WebApps.json
+- Microsoft.Web/stable/2018-02-01/AppServiceEnvironments.json
+- Microsoft.Web/stable/2018-02-01/AppServicePlans.json
+- Microsoft.Web/stable/2018-02-01/ResourceHealthMetadata.json
+directive:
+  # suppress each RPC 3019 error
+- where: $.definitions.Identifier.properties
+  suppress: R3019
+  reason: It's an old API, will resolve in next API version
+  approved-by: "@ravbhatnagar"
+```
+
+### Tag: package-2018-02-only
```

Comment:
Hello. Looks like this tag is same with `package-2018-02`. Is that by design?

Reply:
yes

Reply:
wait

Reply:
yes, it's by design